pom.xml

```diff
@@ -48,6 +48,7 @@
 		<module>org.osgi.test.common</module>
 		<module>org.osgi.test.assertj.framework</module>
 		<module>org.osgi.test.assertj.promise</module>
+		<module>org.osgi.test.assertj.cm</module>
```

Comment:
When we add new projects, we must also add them to the BOM!

Reply:
done